spyder/app/mainwindow.py

```diff
@@ -3047,8 +3047,13 @@ def main():
     # otherwise, optparse won't be able to exit if --help option is passed
     options, args = get_options()
 
+    if options.show_console:
+        print("(Deprecated) --show console does nothing, now the default "
+              " behavior is to show the console, use --hide-console if you "
+              "want to hide it")

```

Comment:
Where this `print` is printed? I'd say is not necessary...

Reply:
It's printed in the console where spyder --show-console is executed

Reply:
Ok, thanks for the clarification!